app/services/push_notification/http_push.rb

```diff
@@ -46,12 +46,7 @@ def deliver_one(service_provider)
         event_type: event.event_type,
         issuer: service_provider.issuer,
       }
-
-      if IdentityConfig.store.risc_notifications_active_job_enabled
```

Comment:
this was the last usage of this feature flag, so now let's fully remove it everywhere by taking it out of `application.yml.default` and `identity_config.rb`

Reply:
This was what I intended with my commit comment -> How -> 3rd bullet. Did you mean something else?

Reply:
Feature flags are safe to remove in the codebase in the same PR that stop using them, so no need for a second PR